core/common/src/main/java/alluxio/Constants.java

```diff
@@ -287,6 +287,10 @@
       "alluxio.worker.filesystem.heartbeat.interval.ms";
   public static final String WORKER_FILE_PERSIST_POOL_SIZE =
       "alluxio.worker.file.persist.pool.size";
+  public static final String WORKER_FILE_PERSIST_RATE_LIMIT_ENABLED =
```

Comment:
As I mentioned in the description, I will update the doc after the new configuration passes the review so I don't need to change the doc back and forth.